### ISSUE TITLE
ParseException message should contain the path of the broken .editorconfig file

### ISSUE THREAD

**Issue by Tapchicoma** [ec4j/ec4j#113]
Whenever `.editorconfig` parsing exception happens, library throws `ParseException`:
```java
Caused by: org.ec4j.core.parser.ParseException: Equals sign '==' missing after property name '' at 1:1 (0)
        at org.ec4j.core.parser.ErrorHandler$3.error(ErrorHandler.java:54)
        at org.ec4j.core.parser.EditorConfigParser.readProperty(EditorConfigParser.java:324)
        at org.ec4j.core.parser.EditorConfigParser.readLine(EditorConfigParser.java:180)
        at org.ec4j.core.parser.EditorConfigParser.readLines(EditorConfigParser.java:148)
        at org.ec4j.core.parser.EditorConfigParser.parse(EditorConfigParser.java:131)
```

Currently there is no way to tell which exactly `.editorconfig` file failed.

Could you add to `ParseException` path to failed file?

Originally reported as a bug in ktlint project: https://github.com/pinterest/ktlint/issues/790 and we could not print more user-friendly message in this case.

**Comment by ppalaga** [comment]
Good catch, thanks! It also raises the question whether empty files should be considered valid. I asked about that here https://github.com/editorconfig/editorconfig/issues/433

**Comment by ppalaga** [comment]
There is now another fix to make the parser accept empty .editorconfig files.
I have just released 0.2.2. You can watch this URL for it to appear in the Central https://repo1.maven.org/maven2/org/ec4j/core/ec4j-core/0.2.2/ec4j-core-0.2.2.pom

**Comment by Tapchicoma** [comment]
Thank you :+1:
